src/pages/TopUp/components/TransactionProgress.tsx

```diff
@@ -111,11 +112,15 @@ const TransactionProgress: React.FC<TransactionProgressProps> = ({
     <Container>
       <div className="flex">
         <ProgressCircle step={1} status={signTxStatus} />
-        <Text className="mt10 ml15">Sign transaction with your wallet</Text>
+        <Text className="mt10 ml15">
+          <FormattedMessage defaultMessage="Sign transaction with your wallet" />
+        </Text>
       </div>
       <div className="mt30 flex">
         <ProgressCircle step={2} status={confirmOnChainStatus} />
-        <Text className="mt10 ml15">Confirm on-chain</Text>
+        <Text className="mt10 ml15">
+          <FormattedMessage defaultMessage="Confirm on-chain" />
```

Comment:
@CarlBeek what does this mean?

Reply:
I'm guessing "Confirm the transaction on-chain", e.g. via a block explorer? Or is this a separate tx?